test/unitTest/user.test.js

```diff
@@ -1,9 +1,10 @@
 
 import chai from 'chai';
-import models from '../../src/models';
+import models from '../../src/models'
```

Comment:
Good job @Daymorelah, but take to heart little details as this
``` suggestion
import models from '../../src/models';
```

Reply:
Thank you. This has been fixed.